packages/remirror__extension-code-block/src/code-block-utils.ts

```diff
@@ -244,12 +244,10 @@ export function getLanguage(props: GetLanguageProps): string {
  */
 export function codeBlockToDOM(node: ProsemirrorNode, extra: ApplySchemaAttributes): DOMOutputSpec {
   const { language, wrap } = omitExtraAttributes(node.attrs, extra);
-  const { style: _, ...extraAttrs } = extra.dom(node);
+  const { ...extraAttrs } = extra.dom(node);
   let style = extraAttrs.style;
 
-  if (wrap) {
-    style = joinStyles({ whiteSpace: 'pre-wrap', wordBreak: 'break-all' }, style);
-  }
+  style = wrap ? joinStyles({ whiteSpace: 'pre-wrap', wordBreak: 'break-all' }, style) : style;
```

Comment:
The style will be added to both the `pre` element and the `code` element. This cause some weird behavior (e.g. The first line of the code block has a padding but other lines don't)
![CleanShot 2023-07-15 at 00 56 21@2x](https://github.com/remirror/remirror/assets/24715727/ec66f99a-b38b-491a-a067-07c2cb8ce68b)

Reply:
I'm not sure but I guess only add `style` to `pre` is good enough?

Reply:
As I mentioned on discord I am trying to avoid touching the code block styling. 

Trying to avoid introducing bugs I will going to revert the changes here.